TripKitDomain/src/main/java/skedgo/tripkit/configuration/Server.kt

```diff
@@ -1,6 +1,6 @@
 package skedgo.tripkit.configuration
 
 enum class Server(val value: String) {
-  Inflationary("https://tripgo.skedgo.com/satapp/"),
+  ApiTripgo("https://api.tripgo.com/v1/"),
```

Comment:
The app name is actually `TripGo` with upper-case `G` instead of `Tripgo`. Can we change it?

Reply:
No problem